### DISCUSSION TITLE
Interactive snapshot mode

### DISCUSSION THREAD

**Discussion by mrspartak** [vitest-dev/vitest#2227] [Ideas]
Basically, the output is some media, that should be previewed and deleted on each test case. 
Jest has it, would be cool to have it in Vitests too. Also do not add a snapshot in the default run. 

I'm trying to find a workaround right now, but I don't see how to get resolvedConfig from Vitests or at least snaphot_update related setting, so I could implement CLI prompts myself in some cases. Will probably go with ENV variables way

**Comment by mrspartak** [comment]
Forget about the second part. We can't interrupt test run and ask something in the console

**Reply by sheremet-va** [reply]
I do like the idea, can you create an issue to implement it?

**Reply by mrspartak** [reply]
Sure, I will

**Reply by mrspartak** [reply]
https://github.com/vitest-dev/vitest/issues/2229
